src/global/CMakeLists.txt

```diff
@@ -7,4 +7,4 @@ set(global_common_files
 add_library(global_common_objs OBJECT ${global_common_files})
 add_library(global STATIC ${libglobal_srcs}
   $<TARGET_OBJECTS:global_common_objs>)
-target_link_libraries(global ceph-common ${DPDK_LIBRARIES} ${EXTRALIBS})
+target_link_libraries(global common ${DPDK_LIBRARIES} ${EXTRALIBS})
```

Comment:
linking statically into global is problematic for targets that link against both global and librados. in src/rgw/CMakeLists.txt, for example:

```cmake
target_link_libraries(rgw_a librados cls_lock_client cls_rgw_client cls_refcount_client
  cls_log_client cls_statelog_client cls_timeindex_client cls_version_client
  cls_replica_log_client cls_user_client ceph-common common_utf8 global
...
```